web/src/components/core/IssuesDialog.jsx

```diff
@@ -78,39 +77,13 @@ const IssueItems = ({ issues = [] }) => {
  *
  * @param {object} props
  * @param {import ("~/client/issues").ClientsIssues} props.issues
+ * @param {string} props.sectionId - A string which indicites which issues section should be highlighted.
  */
 const IssuesSections = ({ issues }) => {
```

Comment:
I'd remove this internal component, no longer needed IMHO. Just put the `<Section>` it holds directly in the place it is going to be rendered. I'd help to simplify the code even more.